near/omni-prover/omni-prover/Cargo.toml

```diff
@@ -26,7 +26,7 @@ container_build_command = ["cargo", "near", "build", "non-reproducible-wasm", "-
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-near-sdk.workspace = true
+near-sdk = { version = "5.15.1", features = ["legacy"] }
```

Comment:
We have to revert this

Reply:
This file has already been deleted after merging with main.